artemVMV/init/index.html

```diff
@@ -4,9 +4,20 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Init</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <title>My path</title>
   </head>
   <body>
-    <div>Hello World</div>
+    <div class="main">
```

Comment:
As far as the class is named main - why not go with the <main> tag?

Reply:
done